error-codes.md

```diff
@@ -133,102 +139,96 @@ In addition, TiDB has the following unique error codes:
+* Error Number: 8040
 
-* Error Number: 8046
-
-    The state of column information is invisible.
+    Unsupported operations are performed. For example, operating lock table on View and Sequence.
 
 * Error Number: 8047
 
     The value of the system variable is not supported. This error usually occurs in the alarm information when the user sets a variable value that is not supported in the database.
 
 * Error Number: 8048
 
-    An unsupported database isolation level is set.
+    An unsupported database isolation level is set. 
+    
+    If you cannot modify the codes by using third-party tools or frameworks, use `tidb_skip_isolation_level_check` to bypass this check.
 
-* Error Number: 8049
+    {{< copyable "sql" >}}
 
-    It fails to load the privilege related table.
+    ```sql
+    set @@tidb_skip_isolation_level_check = 1;
+    ```
 
 * Error Number: 8050
 
     An unsupported privilege type is set.
 
+    See [Privileges required for TiDB operations](/privilege-management.md#privileges-required-for-tidb-operations) for the solution.
+
 * Error Number: 8051
 
-    Unknown field type.
+    Unknown data type is encountered when TiDB parses the Exec argument list sent by the client. 
+    
+    If you encounter this error, check the client. If the client is normal, [contact PingCAP](mailto:info@pingcap.com) for support.
 
 * Error Number: 8052
 
-    The serial number of the data packet from the client is incorrect.
-
-* Error Number: 8053
-
-    An invalid auto-incrementing column value is obtained.
+    The serial number of the data packet from the client is incorrect. 
+    
+    If you encounter this error, check the client. If the client is normal, [contact PingCAP](mailto:info@pingcap.com) for support.
 
 * Error Number: 8055
 
-    The current snapshot is too old. The data may have been garbage collected.
-
-* Error Number: 8056
-
-    Invalid table ID.
-
-* Error Number: 8057
-
-    Invalid field type.
+    The current snapshot is too old. The data may have been garbage collected. You can increase the value of `tikv_gc_life_time` to avoid this problem. The new version of TiDB automatically reserves data for long-running transactions. Usually this error does not occur. 
+    
+    See [garbage collection overview](/garbage-collection-overview.md) and [garbage collection configuration](/garbage-collection-configuration.md).
 
-* Error Number: 8058
+    {{< copyable "sql" >}}
 
-    You apply an automatic variable type that does not exist.
+    ```sql
+    update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_time";
+    ```
 
 * Error Number: 8059
 
-    It fails to obtain an auto-random ID.
+    The auto-random ID is exhausted and cannot be allocated. There is no way to recover from such errors currently. It is recommended to use bigint when using the auto random function to obtain the maximum number of assignment. And try to avoid manually assigning values to the auto random column.
```

Comment:
```suggestion
    The auto-random ID is exhausted and cannot be allocated. There is no way to recover from such errors currently. It is recommended to use bigint when using the auto random feature to obtain the maximum number of assignment. And try to avoid manually assigning values to the auto random column.
```